app/controllers/internal/users_controller.rb

```diff
@@ -47,7 +47,7 @@ def user_status
   def banish
     @user = User.find(params[:id])
     begin
-      Moderator::BanishUser.call_banish(admin: current_user, user: @user)
+      Moderator::BanishUser.call(admin: current_user, user: @user)
```

Comment:
I renamed this method since service objects generally expose only one public method, conventionally called `call`  or `perform`. Having the `_banish` suffix seemed a bit redundant.